### PR TITLE
Update memtotal_darwin.go

### PR DESCRIPTION
**Pull request by urunimi**
Add a comma

### CODE REVIEW

**Comment by urunimi** [comment]
It is not possible to build without the comma.

File: internal/sysinfo/memtotal_darwin.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 
 // PhysicalMemoryBytes returns the total amount of host memory.
 func PhysicalMemoryBytes() (uint64, error) {
-	mib := []int32{6 /* CTL_HW */, 24 /* HW_MEMSIZE */}
+	mib := []int32{6 /* CTL_HW */, 24 /* HW_MEMSIZE */,}
```
Comment (fgrosse) [inline internal/sysinfo/memtotal_darwin.go]: Why wouldn't this build without the comma, which go version do you use?

Comment (urunimi) [inline internal/sysinfo/memtotal_darwin.go]: I am using 1.7.3

Comment (fgrosse) [inline internal/sysinfo/memtotal_darwin.go]: The comma would only [be required][1] if there was a newline before the closing bracket.
What compilation error do you get when building this without the comma?

This is my output:
```
$ go version
go version go1.7.3 linux/amd64

$ GOOS=darwin GOARCH=arm64 go build -v ./internal/sysinfo/memtotal_darwin.go
runtime/internal/sys
runtime/internal/atomic
runtime
internal/race
errors
sync/atomic
sync
syscall
command-line-arguments

$ echo $?                                                                   
0
```

[1]: https://dave.cheney.net/2014/10/04/that-trailing-comma

Comment (urunimi) [inline internal/sysinfo/memtotal_darwin.go]: Oh, it seems that formatter added new lines. Thanks!

Comment (fgrosse) [inline internal/sysinfo/memtotal_darwin.go]: no problem :D

**Pull request closed without merging**